content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -84,11 +84,18 @@ To expand or collapse span descendants, click the chevron (>) icon on a row. To
 {{% /tab %}}
 {{% tab "Map" %}}
 
-{{< img src="tracing/trace_view/map.png" alt="Map" style="width:90%;">}}
+{{< img src="tracing/trace_view/trace-map.png" alt="Trace map" style="width:100%;">}}
```

Comment:
The image is broken, is this a new image that needs to be included?

Reply:
Thanks @mervebolatdd for the links! The image wasn't pulled into this branch, so I merged master into your branch. The image is there now.